wiki/user-reference-guide.md

```
@@ -318,3 +318,125 @@ const error = firestoreRedux.selectors.queriesByRequester({state, requesterId});
+detected as a single `ALL_CAPS_WITH_UNDERSCORES` token (e.g. `IN_PROGRESS`). Unspecified fields
+translate by default (`PLAIN`, `mode: translate`) unless skipped.
+
+### `firestoreRedux.activateTranslation`
+
+Activates translation for a language and a document scope.
+
+```JS
+firestoreRedux.activateTranslation({ id, language, filterFunction });
+```
+
+##### Arguments
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)